bandit/src/lib/query.ts

```diff
@@ -1,11 +1,3 @@
-import type * as AWS from 'aws-sdk';
-import type {
-	GetQueryExecutionOutput, GetQueryResultsOutput,
-	QueryExecutionId, QueryExecutionState,
-	StartQueryExecutionInput,
-} from "aws-sdk/clients/athena";
-import {QueryExecutionNotFoundError, QueryFailedError, QueryPendingError} from "./errors";
-
 export class Query {
```

Comment:
This model isn't necessary anymore, I think we needed the token for athena.
When we call `query()` on the BigQuery client we can actually just pass in a string